src/Frame/TableFrame.py

```diff
@@ -43,6 +46,10 @@ def createTable(self):
         #create tree
         tree = ttk.Treeview(self, columns=dfCols, show='headings')
 
+        #Title for tree
+        titleTemp = 'Summary stats for ' + patientId + "(" + dateLabel + ")"
```

Comment:
this is why you keep it stored as attribute
```suggestion
        titleTemp = 'Summary stats for ' + self.patientId"
```